lib/src/base-styles/_nx-global-header.scss

```diff
@@ -22,6 +23,7 @@
 
   .nx-back-button {
```

Comment:
Back button text currently wraps
![image](https://user-images.githubusercontent.com/3630091/134978604-2a85b2a6-fa57-41e3-9327-c814d0129635.png)

Reply:
> Back button text currently wraps ![image](https://user-images.githubusercontent.com/3630091/134978604-2a85b2a6-fa57-41e3-9327-c814d0129635.png)

Added wrapping to back button text.
Check dca4a3867315315becea917cf424910123da4ee7